qube.cfg

```
@@ -0,0 +1,14 @@
+[bumpversion]
+current_version = 1.3.0-SNAPSHOT
```

```suggestion
current_version = 1.3.0
```
Since this is intended for a release i'd remove the SNAPSHOT from the version
I'd recommend the Qube bump-version functionality for this purpose.